src/test/java/software/amazon/smithy/lsp/SmithyTextDocumentServiceTest.java

```diff
@@ -285,8 +378,46 @@ public void completions() throws Exception {
     }
 
     @Test
-    public void runSelector() throws Exception {
-        Path baseDir = Paths.get(SmithyProjectTest.class.getResource("models").toURI());
+    public void completionsV2() throws Exception {
+        Path baseDir = Paths.get(SmithyProjectTest.class.getResource("models/v2").toURI());
+        String modelFilename = "main.smithy";
+        Path modelMain = baseDir.resolve(modelFilename);
+        List<Path> modelFiles = ListUtils.of(modelMain);
+        try (Harness hs = Harness.create(SmithyBuildExtensions.builder().build(), modelFiles)) {
+            SmithyTextDocumentService tds = new SmithyTextDocumentService(Optional.empty(), hs.getTempFolder());
+            StubClient client = new StubClient();
+            tds.createProject(hs.getConfig(), hs.getRoot());
+            tds.setClient(client);
+            TextDocumentIdentifier mainTdi = new TextDocumentIdentifier(hs.file(modelFilename).toString());
+
+            CompletionParams traitParams = completionParams(mainTdi, 87, 10);
+            List<CompletionItem> traitCompletionItems = tds.completion(traitParams).get().getLeft();
+
+            CompletionParams shapeParams = completionParams(mainTdi, 53,16);
+            List<CompletionItem> shapeCompletionItems = tds.completion(shapeParams).get().getLeft();
+
+            CompletionParams applyStatementParams = completionParams(mainTdi,85, 23);
+            List<CompletionItem> applyStatementCompletionItems = tds.completion(applyStatementParams).get().getLeft();
+
+            CompletionParams whiteSpaceParams = completionParams(mainTdi, 0,0);
```

Comment:
Nit: whitespace

```suggestion
            CompletionParams whiteSpaceParams = completionParams(mainTdi, 0, 0);
```

Reply:
Fixed in [0ed9978](https://github.com/awslabs/smithy-language-server/pull/61/commits/0ed9978ed9b3d58df77fb1af80310514f8d352bd).